llama.cpp

```diff
@@ -1229,7 +1229,8 @@ static llama_vocab::id llama_sample_top_p_top_k(
         }
     }
 
-    sample_top_k(logits_id, top_k);
+    if (top_k > 0 && top_k < n_logits)
+        sample_top_k(logits_id, top_k);
```

Comment:
```suggestion
    if (top_k > 0 && top_k < n_logits) {
        sample_top_k(logits_id, top_k);
    }
```

Reply:
Sorry, I will match to code style later on